### ISSUE TITLE
Binning limits exposures

### ISSUE THREAD

**Issue by robertpascale** [RobboRob/allsky#5]
Binning (mode 2 for 2x2 or mode 4 for 4x4) on the RP HQ Cam will limit exposures taken.
Mode 2 (2x2) will limit exposure to 10.22 seconds
Mode 4 (4x4) will limit exposure to 6.49 seconds
Mode 3 (1x1) will limit exposure to 240 seconds, however, in my testing 240 second exposures actually fail and return a black image. The maximum value that appears to return an image is 200 seconds

**It is worth documenting this**, as effectively only Mode 3 will be of any use for night time all-sky camera usage.

With a fisheye lens, 200 seconds produces visibile trailing, as does 120 seconds.
60 seconds produces trailing, but it is slight, and probably worth attempting for the sensitivity and additional light gathering capability. 30 seconds produces trailing at the extreme outsides of the image, but otherwise pretty good.

**Comment by DarKOrange75** [comment]
Do you think that it is something that could change with updates from Raspberry?

**Comment by DarKOrange75** [comment]
These limitations make the binning unusable for us :-(

**Comment by RobboRob** [comment]
Did you do a rpi-update lately? That solved the issue for me...

**Comment by DarKOrange75** [comment]
Yes but did not check if it solved the issue. Will have a look. Thanks.
